Makefile

```diff
@@ -26,7 +26,7 @@ validate: build
 		observe_domain="$(OBSERVE_DOMAIN)" \
 		location="westus3" \
 		objectId=6bb2971a-0579-49c8-bd59-8d0bcc91ccf9 \
```

Comment:
!!!
are we OK to hard code things like "clientSecretValue" in a public Github repo?

Reply:
probably not, i'll delete this

Reply:
thanks.

There are malicious bots wandering around on Github. parsing public codebase to find credentials.  Back in 2016, in a school project, I pushed my EC2 secret to Github by mistake.  5 minutes later, my AWS account started to create thousands of  EC2 instance across many regions.